### PR TITLE
support paddle generate proposals v2 op

### PR DESCRIPTION
**Pull request by meiyang-intel**
### Details:
 - *Support Paddle generate_proposal_v2 op*

### Tickets:
 - *CVS-73120*

### CODE REVIEW

File: src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp [hunk 1]
```diff
@@ -61,5 +61,56 @@ class OPENVINO_API ExperimentalDetectronGenerateProposalsSingleImage : public Op
...
+        // minimum box width & height
+        float min_size;
+        // specifies NMS threshold
+        float nms_threshold;
+        // number of top-n proposals after NMS
+        int64_t post_nms_count;
+        // number of top-n proposals before NMS
+        int64_t pre_nms_count;
+        // specifies whether the output is dynamic shape or static shape
+        bool dynamic_output = false;
```
Comment (ceciliapeng2011) [inline src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp:84]: Why do you need this "dynamic_output" attribute？

Comment (meiyang-intel) [reply src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp:84]: this is previous design. will change it after the review of spec

File: src/plugins/intel_cpu/src/nodes/mkldnn_experimental_detectron_generate_proposals_single_image_node.cpp [hunk 2]
```diff
@@ -103,18 +106,19 @@ void refine_anchors(const float* deltas, const float* scores, const float* ancho
             proposals[p_idx + 1] = y0;
             proposals[p_idx + 2] = x1;
             proposals[p_idx + 3] = y1;
-            proposals[p_idx + 4] = (min_box_W <= box_w) * (min_box_H <= box_h) * score;
+            proposals[p_idx + 4] = score;
+            proposals[p_idx + 5] = (min_box_W <= box_w) * (min_box_H <= box_h) * 1.0;
```
Comment (vgavrilo) [inline src/plugins/intel_cpu/src/nodes/mkldnn_experimental_detectron_generate_proposals_single_image_node.cpp:110]: We should stay this file as unchanged. But we need an additional file, mkldnn_generate_proposals.cpp/hpp for operation proposed by you.

**Review by vgavrilo — CHANGES_REQUESTED** [review]

File: src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp [hunk 3]
```diff
@@ -61,5 +61,60 @@ class OPENVINO_API ExperimentalDetectronGenerateProposalsSingleImage : public Op
     Attributes m_attrs;
 };
 }  // namespace v6
+
+namespace v8 {
+/// \brief An operation ExperimentalDetectronGenerateProposalsSingleImage
+/// computes ROIs and their scores based on input data.
+class OPENVINO_API ExperimentalDetectronGenerateProposalsSingleImage : public Op {
```
Comment (vgavrilo) [inline src/core/include/openvino/op/experimental_detectron_generate_proposals.hpp:68]: Maybe `GenerateProposals`?

**Review by vgavrilo — CHANGES_REQUESTED** [review]

**Comment by dmitry-gorokhov** [comment]
Closing as non relevant anymore

**Pull request closed without merging**